knowledge-base/common-browser-cache-buster.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Prevent Browser Caching of Telerik CSS and JavaScript Files
+description: Learn how to implement a cache buster for the Telerik UI for Blazor CSS and JavaScript files. Prevent browser caching for the Telerik static NuGet assets.
+type: how-to
+page_title: How to Prevent Browser Caching and Implement a Cache Buster
+slug: common-kb-browser-cache-buster
+position: 
+tags: telerik, blazor
+ticketid: 
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
```

Comment:
Do clients know that they need to implement a cache buster when they come to us? Or do they experience issues and don't know their cause? If it's the latter, you may need to add some of the observed issues to the description.